shopify/session.py

```diff
@@ -28,26 +29,28 @@ def setup(cls, **kwargs):
 
     @classmethod
     @contextmanager
-    def temp(cls, domain, token):
+    def temp(cls, domain, version, token):
         import shopify
-        original_site = shopify.ShopifyResource.get_site()
+        original_domain = shopify.ShopifyResource.url
         original_token = shopify.ShopifyResource.get_headers().get('X-Shopify-Access-Token')
-        original_session = shopify.Session(original_site, original_token)
+        original_version = shopify.ShopifyResource.get_version() or version
+        original_session = shopify.Session(original_domain, original_version, original_token)
 
-        session = Session(domain, token)
+        session = Session(domain, version, token)
         shopify.ShopifyResource.activate_session(session)
         yield
         shopify.ShopifyResource.activate_session(original_session)
 
-    def __init__(self, shop_url, token=None, params=None):
+    def __init__(self, shop_url, version=None, token=None):
```

Comment:
i think it’s fine because it’s not being used, but just calling out that you lost `params` here, in case that was an error

Reply:
Yep, we weren't doing anything with it so I :fire: